apps/main/src/dao/utils/index.ts

```diff
@@ -1,5 +1,6 @@
 import upperFirst from 'lodash/upperFirst'
 import { AlertFormErrorKey, GaugeFormattedData, GaugeMapper } from '@/dao/types/dao.types'
+import { todayInMilliseconds } from '@/dex/utils/utilsDates'
```

Comment:
The build is currently failing because you're importing a `@/dex` file in the dao app. Prob best to consolidate the copy pasted utilsDates from both dex and dao into some shared place like utilsDate in the ui package. Alternatively in ui-kit but I don't mind either way.